brdnet/classifier.py

```diff
@@ -128,34 +135,50 @@ def load_data(args):
     val_Y: numpy.array
         The labels for val_X
     '''
-    Z_df = pandas.read_csv(args.Z_file_path, sep='\t')
+    Z_df = pandas.read_csv(Z_file_path, sep='\t')
 
     # Ensure the gene symbols are in alphabetical order
     Z_df = Z_df.sort_index()
```

Comment:
Just curious, why are you sorting here?

Reply:
Since the script [uses matrix multiplication to transform the training data from gene space to PLIER space](https://github.com/greenelab/brd-net/blob/4831abb34632fabec88a0cdd3d3b747429060763/brdnet/classifier.py#L32), I need to make sure that the genes in `Z_df` and in `train_X/train_Y/val_X/val_Y` are in the same order. 

I get why this is confusing though. I had this call to `sort_index` outside of the `reduce_dimensionality` function for efficiency's sake, but I'll add some code to call it in the function efficiently.